apps/framework-cli/src/infrastructure/kafka_clickhouse_sync.rs

```diff
@@ -287,7 +287,20 @@ fn mapper_json_to_clickhouse_record(
                         };
                     }
                     None => {
-                        record.insert(key, ClickHouseValue::new_string("NULL".to_string()));
+                        // Clickhouse doesn't like NULLABLE arrays so we are inserting an empty array instead.
```

Comment:
Does it not accept `null` in a non-nullable array field, or does it really not like `Nullable(Array(T))`?

Reply:
You can'tt create a table with the schema Array NULLABLE

Reply:
Ah indeed. `Nested type Array(String) cannot be inside Nullable type. (ILLEGAL_TYPE_OF_ARGUMENT)`